src/solver/variable/include/antares/solver/variable/info.h

```
@@ -378,15 +506,29 @@ struct VariableAccessor<ResultsT, Category::dynamicColumns>
         if (*results.isPrinted)
         {
             const Data::PartThermal& thermal = results.data.area->thermal;
+            const auto& shortTermStorage = results.data.area->shortTermStorage;
+            const auto& hydro = results.data.area->hydro;
             for (uint i = 0; i != container.size(); ++i)
             {
-                results.variableCaption = thermal.list.enabledClusterAt(i)->name();
+                vcard_caption_traits<VCardT>::apply(results, i);
 
                 container[i].template buildDigest<VCardT>(results, digestLevel, dataLevel);
             }
         }
     }
 
+    static bool setClusterReserveCaption(SurveyResults& results, uint reserveParticipationIdx)
+    {
+        assert(results.data.area && "Area is NULL");
```

suggestion: assert are not hit in release. If this is a proper error case please throw an exception or handle error properly